helm/dagster/charts/dagster-user-deployments/templates/helpers/_helpers.tpl

```diff
@@ -130,7 +130,7 @@ DAGSTER_K8S_PIPELINE_RUN_ENV_CONFIGMAP: "{{ template "dagster.fullname" . }}-pip
     {{- if .schedulerName }}
     scheduler_name: {{ .schedulerName }}
     {{- end }}
-    namespace: {{ $.Release.Namespace }}
+    namespace: {{ coalesce $.Values.jobNamespace $.Release.Namespace }}
```

Comment:
im not familiar with `coalesce`, you sure this is the right precedence? adding a test would be nice

Reply:
Yeah I switched it. Where would tests live for this, sorry I'm rarely in OSS.

Reply:
https://github.com/dagster-io/dagster/tree/master/helm/dagster/schema/schema_tests

the way we have this set-up i think its easy and quick to add